src/main/java/com/minecolonies/rankup/util/CommonUtils.java

```diff
@@ -50,4 +66,39 @@ public static String timeDescript(int timeNeeded)
         }
         return msg.toString();
     }
+
+    public static String dateNow()
+    {
+        DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
+        java.util.Date today = Calendar.getInstance().getTime();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this fully qualified name with "Date" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1942)